core/validatorapi/router.go

```diff
@@ -71,6 +72,7 @@ func NewRouter(h Handler, beaconNodeAddr string) (*mux.Router, error) {
 		Name    string
 		Path    string
 		Handler handlerFunc
+		Methods []string
```

Comment:
not sure this is required, why not just support any method since it doesn't really matter?

Reply:
yeah, we don't need methods since we use eth2-http package for handler which internally handles methods and stuff.

Reply:
We only want to capture the POST, the GET should be proxied